packages/core/src/common/console-logger-server.ts

```diff
@@ -1,31 +0,0 @@
-/*
- * Copyright (C) 2017 Ericsson and others.
- *
- * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
- */
-
-import { injectable } from "inversify";
-import { LogLevel } from './logger';
-import { ILoggerServer, ILoggerClient } from './logger-protocol';
-
-@injectable()
-export class ConsoleLoggerServer implements ILoggerServer {
-    setLogLevel(name: string, logLevel: number): Promise<void> {
-        return Promise.resolve();
-    }
-    getLogLevel(name: string): Promise<number> {
-        return Promise.resolve(LogLevel.DEBUG);
-    }
-    log(name: string, logLevel: number, message: string, params: any[]): Promise<void> {
-        console.log(`${message} ${params.map(p => p.toString()).join(' ')}`);
```

Comment:
We should map the severity to the respective console methods (trace, debug, log, info, warn, error)

Reply:
OK, let's do that.

Reply:
Did you overlook doing the same thing here, that you now do in `console-logger-server.ts` or is it on purpose?